skops/card/_model_card.py

```diff
@@ -205,20 +206,19 @@ def save(self, path: str | Path) -> None:
 
         card.save(path)
 
-    def _extract_estimator_config(self) -> str:
+    def _write_hyperparameters(self) -> str:
         """Extracts estimator hyperparameters and renders them into a vertical table.
-
         Returns
         -------
         str:
             Markdown table of hyperparameters.
         """
-
         hyperparameter_dict = self.model.get_params(deep=True)
-        table = "| Hyperparameters | Value |\n| :-- | :-- |\n"
-        for hyperparameter, value in hyperparameter_dict.items():
-            table += f"| {hyperparameter} | {value} |\n"
-        return table
+        return tabulate(
+            [[k, v] for k, v in hyperparameter_dict.items()],
```

Comment:
You didn't really reply to that part of my comment, so leaving it here again:

> list(hyperparameter_dict.items())

would also work. I have no preference.

Reply:
ah sorry. sure let me do that, it's more clean.